app/serializers/artist_serializer.rb

```diff
@@ -1,3 +1,4 @@
 class ArtistSerializer < ActiveModel::Serializer
+  has_many :songs, embed: :id, :include => true
```

Comment:
Yep this works!

We usually do things a bit differently, though:  
(1) We specify `embed :ids` so that we don't have to specify it separately for each association.
(2) We usually don't use `include: true` in the serializer.  If we do, that means *every* time we call the `SongSerializer`, we'll get the artists sideloaded.  In some instances, we might not want that.  Instead, you can `include` the artist in your controller.

```ruby
class ArtistSerializer < ActiveModel::Serializer
  embed :ids

  has_many :songs
end

# controller
def index
  # ...
  render json: songs, include: [:artist]
end
```

Also we always use the newer syntax for ruby hashes, which omits the hash rocket:

```
include: true
```